qabelbox/src/main/java/de/qabel/qabelbox/fragments/CreateIdentityMainFragment.kt

```diff
@@ -0,0 +1,92 @@
+package de.qabel.qabelbox.fragments
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import de.qabel.core.repository.exception.EntityExistsException
+import de.qabel.qabelbox.QabelBoxApplication
+import de.qabel.qabelbox.R
+import de.qabel.qabelbox.activities.CreateIdentityActivity
+import de.qabel.qabelbox.identity.interactor.IdentityUseCase
+import org.jetbrains.anko.longToast
+import org.jetbrains.anko.toast
+import javax.inject.Inject
+
+class CreateIdentityMainFragment : BaseIdentityFragment(), View.OnClickListener {
+
+    private var mCreateIdentity: Button? = null
+    private var mImportIdentity: Button? = null
+
+    @Inject
+    internal lateinit var identityUseCase: IdentityUseCase
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        QabelBoxApplication.getApplicationComponent(activity.applicationContext).inject(this)
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view = inflater.inflate(R.layout.fragment_create_identity_main, container, false)
+        mCreateIdentity = view.findViewById(R.id.bt_create_identity) as Button
+        mImportIdentity = view.findViewById(R.id.bt_import_identity) as Button
+        mCreateIdentity!!.setOnClickListener(this)
+        mImportIdentity!!.setOnClickListener(this)
```

Comment:
usually we use kotlinx.android.synthetic to access views in a nice way, i think this is converted code. But if thew view is null after inflate the layout we have bigger problems than a nullpointer...